nimare/generate.py

```diff
@@ -0,0 +1,290 @@
+"""Utilities for generating data for testing"""
+import os
+
+import numpy as np
+import nibabel as nib
+
+from .dataset import Dataset
+from .utils import get_resource_path
+from .meta.utils import compute_ma, get_ale_kernel
+from .transforms import vox2mm, mm2vox
+
+DEF_TINY = np.finfo(float).eps
+
+
+def create_coordinate_dataset(
+    n_foci,
+    fwhm,
+    sample_size,
+    n_studies,
+    foci_coords=None,
```

Comment:
My thought for oversampling came from [Wager, Lindquist, & Kaplan (2007)](https://academic.oup.com/scan/article/2/2/150/2736775), specifically to measure if one study has an outsized impact on the meta-analysis because of the number of peaks it contributes:
> Study 1 (top) contributes 26 peaks to the metaanalysis, many of them very close together, whereas Study 2
(middle) contributes only two.

Reply:
removed `foci_coords` and handle foci generation through the parameter `foci`.